### PR TITLE
configuration should be copied not reused

### PR DESCRIPTION
**Pull request by maggu2810**
Fixes: https://github.com/eclipse/smarthome/issues/6477

### CODE REVIEW

File: bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java [hunk 1]
```diff
@@ -46,7 +59,25 @@ public Configuration() {
...
      */
     public Configuration(Map<String, Object> properties) {
-        this.properties = properties == null ? new HashMap<String, Object>() : ConfigUtil.normalizeTypes(properties);
+        this(properties, false);
+    }
+
+    /**
+     * Create a new configuration.
+     *
+     * @param properties the properties to initialize (my by null)
```
Comment (itn3rd77) [inline bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: Should this be "may be null"?

Comment (maggu2810) [reply bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: fixed

**Comment by htreu** [comment]
Thanks @maggu2810. Waiting for Travis once again (sorry, I did not see the last result).

File: bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ public Configuration(Map<String, Object> properties) {
     /**
      * Create a new configuration.
      *
-     * @param properties the properties to initialize (my by null)
+     * @param properties the properties to initialize (may by null)
```
Comment (itn3rd77) [inline bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: Type only half fixed

Comment (maggu2810) [reply bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: Shame on me

Comment (htreu) [reply bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: And on me :-D

Comment (itn3rd77) [reply bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: And on me for being so niggling :-D

**Pull request merged**